src/_variables.scss

```diff
@@ -0,0 +1,39 @@
+// Colors
```

Comment:
This file can help us standardize styles across the entire app. As it grows, it can be broken up to smaller files.

Reply:
Nice, wasn't sure this would work with scoped vue files. Good to know it does!